src/testFixtures/java/org/opensearch/neuralsearch/TestUtils.java

```diff
@@ -27,15 +27,84 @@
 import org.opensearch.common.xcontent.XContentHelper;
 import org.opensearch.core.common.bytes.BytesReference;
 import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.knn.index.SpaceType;
 import org.opensearch.search.SearchHit;
 import org.opensearch.search.SearchHits;
 import org.opensearch.search.fetch.FetchSearchResult;
 import org.opensearch.search.query.QuerySearchResult;
 
 public class TestUtils {
 
-    private static final String RELATION_EQUAL_TO = "eq";
+    public static final String RELATION_EQUAL_TO = "eq";
     public static final float DELTA_FOR_SCORE_ASSERTION = 0.001f;
+    public static final String CLIENT_TIMEOUT_VALUE = "90s";
```

Comment:
I think most of the constants that were moved here belong to exact integ test classes. In this shared util class we need to have something very generic and/or applicable to multiple tests, like ` NODES_BWC_CLUSTER` or `UPGRADED_CLUSTER`.